app/infrastructure/cache/cache_service.py

```
@@ -16,7 +16,8 @@ async def set(key: str, value: Any, expire: int | None = None, serialize: bool =
 				value = json.dumps(value)
 			await client.set(key, value, ex=expire)
 			return True
-		except Exception:
+		except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)